setup.cfg

```diff
@@ -0,0 +1,210 @@
+[flake8]
+exclude =
+    # The following list should eventually be empty.
+    examples/tutorials/doc_examples/examples,
+    examples/tutorials/traitsui_4.0,
```

Comment:
If I try to list all individual files, flake8 takes a long time to run doing the filtering.
So I took the liberty to exclude an entire directory, by taking into account what are frequently changed together in a single PR, and the proportion of uncleaned files in a directory.